lxd/openfga/model.go

```diff
@@ -0,0 +1,4 @@
ork","relations":{"project":{"this":{}},"manager":{"union":{"child":[{"this":{}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"network_manager"}}}]}},"viewer":{"union":{"child":[{"this":{}},{"computedUserset":{"object":"","relation":"manager"}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"network_viewer"}}}]}}},"metadata":{"relations":{"project":{"directly_related_user_types":[{"type":"project"}]},"manager":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]},"viewer":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]}}}},{"type":"network_acl","relations":{"project":{"this":{}},"manager":{"union":{"child":[{"this":{}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"network_acl_manager"}}}]}},"viewer":{"union":{"child":[{"this":{}},{"computedUserset":{"object":"","relation":"manager"}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"network_acl_viewer"}}}]}}},"metadata":{"relations":{"project":{"directly_related_user_types":[{"type":"project"}]},"manager":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]},"viewer":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]}}}},{"type":"network_zone","relations":{"project":{"this":{}},"manager":{"union":{"child":[{"this":{}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"network_zone_manager"}}}]}},"viewer":{"union":{"child":[{"this":{}},{"computedUserset":{"object":"","relation":"manager"}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"network_zone_viewer"}}}]}}},"metadata":{"relations":{"project":{"directly_related_user_types":[{"type":"project"}]},"manager":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]},"viewer":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]}}}},{"type":"storage_volume","relations":{"project":{"this":{}},"manager":{"union":{"child":[{"this":{}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"storage_volume_manager"}}}]}},"viewer":{"union":{"child":[{"this":{}},{"computedUserset":{"object":"","relation":"manager"}},{"tupleToUserset":{"tupleset":{"object":"","relation":"project"},"computedUserset":{"object":"","relation":"storage_volume_viewer"}}}]}}},"metadata":{"relations":{"project":{"directly_related_user_types":[{"type":"project"}]},"manager":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]},"viewer":{"directly_related_user_types":[{"type":"user"},{"type":"group","relation":"member"}]}}}}],"schema_version":"1.1"}`
```

Comment:
where does this come from?

Reply:
There's a tool which transforms the DSL to JSON and vice versa: https://github.com/openfga/syntax-transformer#cli

Reply:
Can we integrate that into the Makefile like we've done with other generated files?

Reply:
Yeah something like that would be good, because then it will make it easier for others to update in the future.
We should also add a check to ensure that if the policy changes and the file is not updated then the tests fail (like we do for the po files).